modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc

```diff
@@ -110,34 +110,30 @@ from kfp import dsl
 )
 
 
-def ray_fn(openshift_server: str, openshift_token: str):
+def ray_fn():
    import ray <1>
-   from codeflare_sdk import Cluster, ClusterConfiguration, TokenAuthentication, generate_cert
+   import time <2>
+   from codeflare_sdk import Cluster, ClusterConfiguration, generate_cert <3>
 
 
-   auth = TokenAuthentication( <2>
-       token=openshift_token, server=openshift_server
-   )
-   auth_return = auth.login()
-
-
-   cluster = Cluster( <3>
+   cluster = Cluster( <4>
        ClusterConfiguration(
+           namespace="my_project", <5>
            name="raytest",
            num_workers=1,
            head_cpus="500m",
            min_memory=1,
            max_memory=1,
            num_gpus=0,
-           image="quay.io/project-codeflare/ray:latest-py39-cu118", <4>
-           local_queue="local_queue_name", <5>
+           image="quay.io/project-codeflare/ray:latest-py39-cu118", <6>
+           local_queue="local_queue_name", <7>
        )
    )
 
 
    print(cluster.status())
-   cluster.up() <6>
-   cluster.wait_ready() <7>
+   cluster.up() <8>
+   time.sleep(180) <9>
```

Comment:
Should we say the reason behind time.sleep? that is a kwon issue

Reply:
Added for now, but we don't usually document known issues in the main user doc.